pkg/config/config.go

```diff
@@ -221,9 +221,9 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("log_file", "")
 	config.BindEnvAndSetDefault("log_file_max_size", "10Mb")
 	config.BindEnvAndSetDefault("log_file_max_rolls", 1)
-	config.BindEnvAndSetDefault("log_level", "info")
+	config.BindEnvAndSetDefault("log_level", "info", "DD_LOG_LEVEL", "LOG_LEVEL")
```

Comment:
isn't `DD_LOG_LEVEL` automatically created by `BindEnvAndSetDefault` ? Same for `DD_LOG_TO_CONSOLE`

Reply:
Yeah good point